prodactivity/helpers/FetchHelper.ts

```diff
@@ -0,0 +1,115 @@
+// INTERFACES
+interface GetParams {
+    [index: string]: any
+}
+
+interface JSONResponse {
+    [index: string]: any
+}
+
+
+// HELPER FUNCTIONS
+
+// Actually makes the fetch call and throws an Error if our status code is bad
+const _api = async (url: string, options: RequestInit): Promise<JSONResponse | undefined> => {
+
+    const encodedURL: string = encodeURI(url);
+
+    const response: Response = await fetch(encodedURL, options);
+    if (!response.ok) {
+        console.log(response.statusText);
+        console.log(response.status);
```

Comment:
not sure what information is conveyed with these two console logs but we should be careful what we console out

Reply:
I can remove them, these are leftover from when FetchHelper was just a util from my capstone project.

Reply:
We could keep them and then wrap them in an if statement for lower environments only